src/templates/base/2019/index.html

```diff
@@ -36,11 +36,11 @@ <h2>{{ self.intro_sub_title() }}</h2>
         {{ self.start_exploring() }}
       </a>
     </div>
-    <div class="intro-image-year-wrapper">
-      <div class="intro-image-year"></div>
-    </div>
     <div class="intro-image-wrapper">
-      <div class="intro-image"></div>
+      <svg viewBox="0 0 42 16">
+        <text x="0" y="15">{{ year }}</text>
+      </svg>      
+      <img src="/static/images/home-hero.png" alt="">
```

Comment:
Do we really need to use SVG for simple text?

I notice this breaks on IE11 as it doesn't seem to calculate the height of the SVG correctly. Not that we have much traffic from this but still that, along with the magic numbers required, are a downside to this approach.

If you changed to this:

```suggestion
      <div class="year">2019</div>     
      <img src="/static/images/home-hero.png" alt="">
```

Along with this CSS:

```css
.intro-image-wrapper .year {
  color: #f7f779;
  font-family: 'Poppins', sans-serif;
  font-weight: 700;
  font-size: 12rem;
  line-height: 0.58;
  margin-top: 0.2em;
}
```

We seem to get the same effect (though need slightly different numbers for mobile).

WDYT?

Reply:
As you already know "mobile" breakpoint is hard to define and our current max-width:900px breakpoint might not be enough.

I am aware of the method you're showcasing above, but I avoided as you need to define different font sizes across different media queries breakpoints.

Made some research and I preferred using SVG because it preserves aspect ratio: https://css-tricks.com/fitting-text-to-a-container/#just-use-svg. I also add a CSS fix for the IE11 only. It degrades nicely now in my opinion.

Reply:
Ah OK. So your method allows the image to change size responsively and for the text to automatically resize as necessary (like happens between 900px and 1050px where image shrinks slightly slightly). OK happier now there is a reason. Resolving this.

And thanks for the IE11 hack. Not loving it as so IE specific, but will still take it. Hopefully soon enough I'll learn to get over supporting IE11 but since you've found the solution let's take it for now.